starknet_py/tests/e2e/fixtures/misc.py

```diff
@@ -13,6 +14,10 @@
     CONTRACTS_COMPILED_V0_DIR,
     CONTRACTS_COMPILED_V1_DIR,
     CONTRACTS_COMPILED_V2_DIR,
+    CONTRACTS_V1_ARTIFACTS,
+    CONTRACTS_V1_COMPILED,
```

Comment:
what is the difference between `artifacts` and `compiled`? I though artifacts are just compiled contracts?

Reply:
`starknet_artifacts.json` contains information about all compiled contracts in Scarb project, use it to get paths to my contract by they names

Reply:
This file is a map, that resides in target directory. This still does not explain why you need `CONTRACTS_V1_ARTIFACTS` and `CONTRACTS_V1_COMPILED`

Reply:
```
CONTRACTS_V2_COMPILED = MOCK_DIR / "contracts_v2/target/dev"
CONTRACTS_V2_ARTIFACTS = CONTRACTS_V2_COMPILED / "contracts_v2.starknet_artifacts.json"

CONTRACTS_V1_COMPILED = MOCK_DIR / "contracts_v1/target/dev"
CONTRACTS_V1_ARTIFACTS = CONTRACTS_V1_COMPILED / "contracts_v1.starknet_artifacts.json"
```

It seems to me to be cleaner.  `contracts_v1.starknet_artifacts.json` the project name is included in this starknet_artifacts file name